docs/src/modules/components/DemoFrame.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { create } from 'jss';
-import { withStyles, jssPreset } from '@material-ui/core/styles';
-import { StylesProvider } from '@material-ui/styles';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
I thought the old module was removed and we use
```diff
- import { withStyles } from '@material-ui/core/styles';
+ import { withStyles } from '@material-ui/styles';
```

Reply:
You made me think that I have forgotten the hook demos. I have done the following change:
```diff
-import { makeStyles } from '@material-ui/styles';
+import { makeStyles } from '@material-ui/core/styles';
```
So people can copy & paste the demos directly.